librz/core/cmd_linux_heap_glibc.c

```diff
@@ -1,3 +1,6 @@
+// SPDX-FileCopyrightText: Pulak Malhotra
```

Comment:
@PulakIIIT do you want to put your email here?

Reply:
```suggestion
// SPDX-FileCopyrightText: Pulak Malhotra <pulakmalhotra2000@gmail.com>
```